### DISCUSSION TITLE
Lots of Astra / TVHeadend server streams

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#96] [Show and tell]
I have found lots of Astra service playlists and I'm working on adding all of those in this repo.
Since there's LOTS of channels (lots of them are actually from Nilesat), I thought about sharing those with you.

All of the links mentioned in this post are verified by myself.

EDIT 17/12/21 : New version, rebuilding list since it disappeated from Ghostbin

[Link](https://telegra.ph/Astra-12-17)

**Comment by Dum4G** [comment]
http://94.236.145.205:8000/playlist.m3u8 Vivacom Bulgaria

**Reply by BellezaEmporium** [reply]
Getting a server timeout on that one.

**Reply by Dum4G** [reply]
Works immediately in VLC to me

**Reply by BellezaEmporium** [reply]
> Works immediately in VLC to me

Maybe there's some internal geolock or my connection isn't good x)

**Reply by Dum4G** [reply]
What if you try a single channel?
http://94.236.145.205:8000/play/a00f

**Reply by BellezaEmporium** [reply]
Same issue. ERR_TIMED_OUT

**Reply by BellezaEmporium** [reply]
Ah, works on another connection.

**Reply by BellezaEmporium** [reply]
![image](https://user-images.githubusercontent.com/30985701/121500933-098c1f00-c9df-11eb-861c-25f0e625a7f4.png)

**Comment by MapGuy11** [comment]
> I have found lots of Astra service playlists and I'm working on adding all of those in this repo.
> Since there's LOTS of channels (lots of them are actually from Nilesat), I thought about sharing those with you.
> 
> All of the links mentioned in this post are verified by myself.
> 
> Ghostbin link (https://ghostbin.com/paste/Z3C1e/LaneSh4d0w)

Any US Channels?

**Reply by BellezaEmporium** [reply]
Might have some... I know there's some Latin variants, but there might be some original channels.

**Reply by BellezaEmporium** [reply]
I have some original UK channels such as 
http://46.4.119.218:8880/play/a05t (BBC Entertainment)
http://46.4.119.218:8880/play/a07t (EDGEsport (bad frames))

However, I have some AMC, AXN etc etc...

http://46.4.119.218:8880/play/a03v CNBC Europe

And... I even have French channels, like MCM Pop : http://46.4.119.218:8880/play/a03s

**Comment by joaquinito2048** [comment]
Hello, when all Astra / TVHeadend streams will be added finally???

**Reply by BellezaEmporium** [reply]
Hello, we'll need to sort all channels out and add them on the different countries. We've already started to do so not so long ago.

**Comment by BellezaEmporium** [comment]
http://196.179.251.77:8877/playlist.m3u8 - Canalsat France
http://195.35.85.115:8000/playlist.m3u8 - Globesat Worldwide
http://93.109.224.178:8001/playlist.m3u8 - IRIB Iran
http://14.199.164.20:4001/playlist.m3u8 - ? India (big list)
http://103.157.237.18:8000/playlist.m3u8 - Dinajpur Vision Bangladesh
http://71.105.16.202:8002/playlist.m3u8 - Sounds like an USA satellite (Hotbird ?) but can't seem to put my hand on it. No data.
http://185.188.190.170:8000/playlist.m3u8 - Astra Czech Republic
http://149.62.177.149:1980/playlist.m3u8 - Movistar + Spain (other set of channels) (watch out, there's Playboy on this one)

**Reply by MapGuy11** [reply]
http://71.105.16.202:8002/playlist.m3u8 - Primarily NYC / National US Feeds Also Indian stuff

**Reply by BellezaEmporium** [reply]
> 
> 
> http://71.105.16.202:8002/playlist.m3u8 - Primarily NYC / National US Feeds Also Indian stuff

Yeah, the fact that there's specific national channels makes me think it's based on an american satellite.

**Reply by MapGuy11** [reply]
Though most are marked wrong lol

**Reply by BellezaEmporium** [reply]
Ah well lol, at least, it works (tm)

**Reply by MapGuy11** [reply]
Did you see my discussion post? I went and found a lot of US Control Panels.

**Reply by MapGuy11** [reply]
I have no idea if they work or not

**Reply by BellezaEmporium** [reply]
I've seen your first one, and then I just found the second one. Those are interesting...

**Reply by MapGuy11** [reply]
Cool :+1:

**Reply by SSyar** [reply]
Pakistan ?

**Reply by BellezaEmporium** [reply]
> 
> 
> Pakistan ?

I don't think i've seen anything from Pakistan yet... don't they stream on Badr 3 ?

**Reply by SSyar** [reply]
> 
> 
> > Pakistan ?
> 
> I don't think i've seen anything from Pakistan yet... don't they stream on Badr 3 ?

i ve no idea

**Comment by Sphinxroot** [comment]
NB2HI4DTHIXS6Z3JORUHKYROMNXW2L2GOJSWKLKJKBKFML2DN52W45DSNFSXGL3JONZXKZLTF42DENA=

**Reply by BellezaEmporium** [reply]
Is this the password of something ?

**Reply by Sphinxroot** [reply]
nop  hint devide by 2

**Reply by BellezaEmporium** [reply]
I see. Specific channels indeed. It's a bit "touchy" when we talk about those. 

For the DMCAs... we can't be blamed for referencing publically available links. The fact that we *found them* being hosted is different from *hosting them*. I guess it's a wait and see.

Of course, if the copyright holders wants the link off our playlists we'll delete them. But well, better take off the root issue of the problem instead of deleting references of that root issue x)

**Comment by heroku-miraheze** [comment]
@LaneSh4d0w Are you adding more channels for ASTRA Sat and for TVHeadend???

**Reply by BellezaEmporium** [reply]
Not for the moment. I am focusing on other things.

**Reply by BellezaEmporium** [reply]
There you go : http://91.218.211.8:8080/playlist.m3u8 | Polsat (incl. 8 adult channels and various other programs)

**Comment by heroku-miraheze** [comment]
I sure, yes.

**Reply by SSyar** [reply]
??? ??

**Comment by Sphinxroot** [comment]
Will write here tested links (LaneSh4d0w) :

http://45.92.69.244:9000/playlist.m3u8 (Italian - Forbidden on my side)
http://37.201.24.32:8000/playlist.m3u8 (no playlist found)
http://31.177.62.174:8000/playlist.m3u8 (Greek (incl. radio) - works)
http://31.177.62.174:8001/playlist.m3u8 (Greek - works)
http://31.177.62.174:8002/playlist.m3u8 (Greek - works)
http://31.177.62.174:8003/playlist.m3u8 (Greek - works)
http://213.162.202.5:4000/playlist.m3u8 (Middle East - works)
http://190.92.72.90:8005/playlist.m3u8 (doesn't seem to answer...)
http://181.115.72.65:8099/playlist.m3u8 (Latin America (Colombia it seems ?) - works)
http://190.7.192.165:8001/playlist.m3u8 (only one channel that doesn't seem to work correctly)
http://186.67.37.45:9000/playlist.m3u8 (Argentinian - works)
http://161.132.108.34:20000/playlist.m3u8 (Mix of Spanish and Latin American channels - works)
http://80.251.50.18:8004/playlist.m3u8 (Russian / Bulgarian channels - works)
http://201.168.205.12:8000/playlist.m3u8 (Latin America (Colombia it seems ?), incl. adult channels - works)
http://170.82.16.234:9981/playlist.m3u8 (Brazilian - Forbidden on my side)
http://194.146.190.241:4000/playlist.m3u8 (Ukrainian / Russian channels - works)

**Comment by SSyar** [comment]
http://103.153.39.34:8000/playlist.m3u8 found this., dont know any association

**Reply by BellezaEmporium** [reply]
> 
> 
> http://103.153.39.34:8000/playlist.m3u8 found this., dont know any association

Indian channels from what I see. Poor bandwidth too sadly.

**Comment by SSyar** [comment]
http://103.17.201.18:8000/playlist.m3u8 - 1st channel not working

**Comment by SSyar** [comment]
is there any way to find these playlists other than wildcard search on google or sites ?

**Reply by heroku-miraheze** [reply]
Yes, go to https://iptvcat.com/home_11, and there, you can find all channels and add it to our playlist. Good luck!!!

**Reply by BellezaEmporium** [reply]
> 
> 
> Yes, go to https://iptvcat.com/home_11, and there, you can find all channels and add it to our playlist. Good luck!!!

This one is basically a scraper, it sometimes also takes links from our repo and some others.

**Reply by BellezaEmporium** [reply]
> 
> 
> is there any way to find these playlists other than wildcard search on google or sites ?

Apart of saying "Astra Control Panel" when searching on Google, maybe use something like Censys.io

![image](https://user-images.githubusercontent.com/30985701/138563720-f4c6dd3e-1e80-4c0c-a544-acde42d7a023.png)

**Reply by SSyar** [reply]
i better stick to., ghostbin :D

**Reply by KingMorisco** [reply]
thanks bro , but tell me i can found astra control panel from this search ?

**Reply by BellezaEmporium** [reply]
> thanks bro , but tell me i can found astra control panel from this search ?

The image is pretty much self-explanatory. Once you've done that, all IPs that are given by Censys contains 1 or at least 1 Astra Control Panel (for each port)

**Reply by KingMorisco** [reply]
yes thank you, another question if you plese about how i can  fin user and password of astra panel
![Screenshot_1](https://user-images.githubusercontent.com/65326054/169829166-acedda80-52e1-41da-974f-13d577ac634e.png)

**Comment by BellezaEmporium** [comment]
http://109.98.247.162:9999/ Don't ask me how I got that. TVR Craiova + Teletext

**Comment by BellezaEmporium** [comment]
http://93.125.121.108:8284/playlist.m3u8 -> Russian channels [Origin: Belarus]
http://178.172.225.191:8284/playlist.m3u8 -> Same host [Origin: Belarus]
http://45.232.208.25:8000/playlist.m3u8 -> Typical set of LATAM channels, the one I've used to see on the other things. [Origin: Chile]
http://45.232.208.164:8000/playlist.m3u8 -> Same host [Origin: Chile]

**Comment by BellezaEmporium** [comment]
Ghostbin updated.

**Comment by Johnny-c-Max** [comment]
Some links respond with 403 error eg 185.99.253.46:8000 although there is still a playlist there.
But where are these links from, are there some IPTV services based on Astra or TVHeadend?

**Comment by BellezaEmporium** [comment]
http://45.232.178.10:8000/playlist.m3u8 <- LATAM (MX / CL)

**Comment by Johnny-c-Max** [comment]
Some European channels but it seems like some of the links stop frequently…

http://83.78.23.248:7000/playlist.m3u8

**Comment by BellezaEmporium** [comment]
lol, all the links disappeared, 404 on Ghostbin.

**Comment by BellezaEmporium** [comment]
http://45.236.175.203:9600/playlist.m3u8 Chile
http://45.7.135.121:58002/playlist.m3u8 Colombia
http://45.7.135.123:58001/playlist.m3u8 Colombia follow up link above
